packages/components/src/lib/controllers/app-controller.js

```diff
@@ -377,7 +384,11 @@ export default class AppController {
       buildNumber: this._options.app.buildNumber,
       storageAdapter: this._options.storageAdapter
     })
-    // All options has been loaded and initialized after this point
+    // All options are initialized at this point
+
+    // We do have dependency on the DataModelController only because we're using local fields
+    // with resolvers from the DataModelController. This dependency will not exist with the remote GraphQL service.
```

Comment:
Even when we have a remote GraphQL service I expect we will also have a client-side service as well (e.g. for treebank and other client-side data)

Reply:
Yes, that's right, I'll change the comment to reflect that.